public/js/main.js

```diff
@@ -23,9 +23,16 @@ calendar.render();
 addDateEventListeners();
 
 // event listener for calendar button in header
-document.getElementById('calendar-button').addEventListener('click', function () {
-  document.getElementById('list-page-content').classList.toggle('hide');
-  document.getElementById('show-cal').classList.toggle('show');
+document.getElementById('calendarIcon').addEventListener('click', function () {
+  var list = document.getElementById('list-page-content');
+  var calendar = document.getElementById('show-cal');
+  if (list.style.display === 'block') {
+    list.style.display = 'none';
+    calendar.style.display = 'block';
+  } else {
+    calendar.style.display = 'none';
+    list.style.display = 'block';
+  }
```

Comment:
It's better to do this the way you were originally doing it - by toggling a class. Was there a reason you changed this?

Reply:
I think the toggle is for framework , like bootstrap, tachyons
we didn't want to use bootstrap

Reply:
You can do it with vanilla js, just need to create a class called `hide` that has `display: none` and toggle that on or off

Reply:
This still needs changing back